src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -771,24 +777,48 @@ const WorkflowView = _.flow(
             }, ['Run analysis'])
           }),
           activeTab === 'outputs' && !currentSnapRedacted && div({ style: { marginBottom: '1rem' } }, [
-            div({ style: styles.outputInfoLabel }, 'Output files will be saved to'),
-            div({ style: { display: 'flex', alignItems: 'center' } }, [
-              div({ style: { flex: 'none', display: 'flex', width: '1.5rem' } }, [icon('folder', { size: 18 })]),
-              div({ style: { flex: 1 } }, [
-                'Files / ',
-                span({ style: styles.placeholder }, 'submission unique ID'),
-                ' / ', wdl ? wdl.match(/^\s*workflow ([^\s{]+)\s*{/m)[1] : span({ style: styles.placeholder }, 'workflow name'), ' / ',
-                span({ style: styles.placeholder }, 'workflow unique ID')
-              ])
-            ]),
-            !!rootEntityType && h(Fragment, [
-              div({ style: { margin: '0.5rem 0', borderBottom: `1px solid ${colors.dark(0.55)}` } }),
-              div({ style: styles.outputInfoLabel }, 'References to outputs will be written to'),
-              div({ style: { display: 'flex', alignItems: 'center' } }, [
-                div({ style: { flex: 'none', display: 'flex', width: '1.5rem' } }, [icon('listAlt')]),
-                `Tables / ${rootEntityType}`
+            h(Collapse, { title: 'Output storage' }, [
+              h(Fragment, [
+                div({ style: styles.outputInfoLabel }, 'Output files will be saved to your workspace bucket:'),
+                div({ style: { display: 'flex', alignItems: 'center', margin: '0.5rem' } }, [
+                // TODO: why can't I use the 'RegionFlaggedContent' element here...?
+                  div({ style: { flex: 'none', display: 'flex', width: '1.5rem' } }, [
+                    h(TooltipTrigger, { content: div([`${locationType}: ${location} (${bucketRegionDescription})`]) }, [div(bucketRegionFlag)])
```

Comment:
If this tooltip contains important information, then we should include some accessible text so screen readers can see it as well.